docs/release-notes/highlights-1.2.0.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[release-highlights-1.2.0]]
+== 1.2.0 release highlights
+
+[float]
+[id="{p}-120-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 1.2.0 of {n}. See <<release-notes-1.2.0>> for the full list of changes.
+
+[float]
+[id="{p}-120-enterprise-search"]
+==== Support for Elastic Enterprise Search
+
+ECK 1.2.0 introduces a new Custom Resource Definition (CRD) to easily deploy and orchestrate link:https://www.elastic.co/enterprise-search[Elastic Enterprise Search] on Kubernetes. With just a few lines of YAML configuration, you can launch an instance of link:https://www.elastic.co/app-search/[App Search] or link:https://www.elastic.co/workplace-search[Workplace Search] (requires an Enterprise license or a <<{p}-licensing,trial>>) and connect it to an Elasticsearch cluster.
+
+
+[float]
+[id="{p}-120-beats"]
+==== Support for Beats
+
+Deploying link:https://www.elastic.co/beats/[Beats] on Kubernetes is now much easier using the new Beats CRD. You can deploy and manage data shippers such as Filebeat, Metricbeat, Auditbeat and others using ECK and effortlessly connect them to Elasticsearch and Kibana for storage and analysis.
```

Comment:
We repeat the "is easier" phrasing here and in the APM agent configuration, I wonder if there is a way to make it more varied

Reply:
How about "is more straightforward"?